src/cart/crt_rpc.h

```diff
@@ -662,4 +662,6 @@ int crt_iv_sync_corpc_pre_forward(crt_rpc_t *rpc, void *arg);
 int
 crt_proto_register_internal(struct crt_proto_format *crf);
 
+char *crt_opc_to_str(crt_opcode_t opc);
```

Comment:
char *
crt_opc_to_str(crt_opcode_t opc);

with a short doxygen comment would not hurt :)

Reply:
I will add a comment, but as a note, this is an internal function not visible to the end-user. We do not generate doxygen documentation for our internal headers